### PR TITLE
Clarified format of postgres_dsn

### PR DESCRIPTION
**Pull request by cwarnermm**
Based on customer feedback that ``--target "${POSTGRES_DSN}"`` wasn't sufficient, and ``postgres://${POSTGRES_DSN}`` would be clearer instruction in the docs that leads admins to successful outcomes.

### CODE REVIEW

File: source/deploy/postgres-migration.rst [hunk 1]
```diff
@@ -275,9 +275,13 @@ For our case, we can simply run the following command:
 
 .. code:: sh
 
-   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN}" --exclude="db_migrations,ir_,focalboard,systems"
+   dbcmp --source "${MYSQL_DSN}" --target "postgres://${POSTGRES_DSN} " --exclude="db_migrations,ir_,focalboard,systems"
```
Comment (isacikgoz) [inline source/deploy/postgres-migration.rst:278]:
```suggestion
   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN} " --exclude="db_migrations,ir_,focalboard,systems"
```

File: source/deploy/postgres-migration.rst [hunk 2]
```diff
@@ -455,7 +459,7 @@ Compare the plugin data
 
 .. code:: sh
 
-   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN}" --exclude="db_migrations,systems"
+   dbcmp --source "${MYSQL_DSN}" --target "postgres://${POSTGRES_DSN}" --exclude="db_migrations,systems"
```
Comment (isacikgoz) [inline source/deploy/postgres-migration.rst:462]:
```suggestion
   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN}" --exclude="db_migrations,systems"
```

**Review by isacikgoz — COMMENTED** [review]
I'd recommend documenting like below.

File: source/deploy/postgres-migration.rst [hunk 3]
```diff
@@ -275,9 +275,13 @@ For our case, we can simply run the following command:
...
 
-   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN}" --exclude="db_migrations,ir_,focalboard,systems"
+   dbcmp --source "${MYSQL_DSN}" --target "postgres://${POSTGRES_DSN} " --exclude="db_migrations,ir_,focalboard,systems"
 
-Note that this migration guide only covers the tables for Mattermost products.
+For example: ``dbcmp --source "user:password@tcp(address:3306)/db_name --target "postgres://user:password@address:5432/db_name``
+
+.. note::
+   
+   This migration guide only covers the tables for Mattermost products.
```
Comment (isacikgoz) [inline source/deploy/postgres-migration.rst:284]:
```suggestion
An example command would look like: ``dbcmp --source "user:password@tcp(address:3306)/db_name --target "postgres://user:password@address:5432/db_name``

.. note::
   
   ``POSTGRES_DSN`` should start with a ``postgres://`` prefix. This way ``dbcmp`` decides which driver to use while connecting to a database.
```

**Review by isacikgoz — COMMENTED** [review]
This is another suggestion, somehow didn't appear on my previous comment.

**Comment by cwarnermm** [comment]
Thanks for your guidance, @isacikgoz!

**Pull request merged**